app/styles/addon-overrides/_ember-data-table.scss

```diff
@@ -1,3 +1,7 @@
 td, th {
   border-radius: 0;
 }
+
+.auk-table > *[data-table], .auk-table .data-table-content {
+  overflow: visible;
+}
```

Comment:
@brenner-company hier de fix om ervoor te zorgen dat de dropdown niet voor een overflow zorgt in een `<DataTable>`.